MERN Scripts/mern-emsystem/src/components/LocationSelect.js

```diff
@@ -0,0 +1,29 @@
+
+import React, { useState } from 'react';
+
+const LocationSelect = () => {
+  const [selectedLocation, setSelectedLocation] = useState('Select a location');
+
+  const handleChange = (event) => {
+    setSelectedLocation(event.target.value);
+    localStorage.setItem('selectedLocation', event.target.value);
+    window.location.reload();
+  }
+
+  return (
+    
+    <select value={selectedLocation} onChange={handleChange} >
+      <option value="Rutland">Select a location</option>
+      <option value="West Kelowna">West Kelowna</option>
+      <option value="Rutland">Rutland</option>
+      <option value="Mission">Mission</option>
+      <option value="Lake Country">Lake Country</option>
+      <option value="Glenmore"> Glenmore </option>
+      <option value="Kelowna Central"> Kelowna Central </option>
+    </select>
```

Comment:
we should retrieve this from the Location Collection in the DB, can we do that before merging? i.e. map the Location.Name values for the options

Reply:
I will make an issue for this , will make another pr for this